lib/matplotlib/axes/_axes.py

```diff
@@ -3194,6 +3194,10 @@ def grouped_bar(self, heights, *, positions=None, group_spacing=1.5, bar_spacing
 
         **kwargs : `.Rectangle` properties
 
+            The following properties additionally accept a sequence of values
+            corresponding to the datasets in *heights*:
+            *edgecolor*, *facecolor*, *linewidth*, *linestyle*, *hatch*.
```

Comment:
Should we make this a bit more prominent?

```suggestion
            Properties applied to all bars. The following properties additionally
            accept a sequence of values corresponding to the datasets in
            *heights*:
            
            - *edgecolor*
            - *facecolor*
            - *linewidth*
            - *linestyle*
            - *hatch*
```